src/components/Boards/Board/Board.tsx

```diff
@@ -0,0 +1,100 @@
+import { Component, ReactNode } from "react";
+import { connect } from "react-redux";
+import "components/Boards/Board/Board.css";
+import { AppDispatch, RootState } from "store/store";
+import { IBoard, IIssue, IUser } from "common/models";
+import BoardColumn from "components/Boards/Board/BoardColumn/BoardColumn";
+import { Grid, Avatar, Divider } from "@material-ui/core";
+
+interface BoardProps {
+  groupBy: string;
+  selectedBoard: string;
```

Comment:
This can say what property of the board we are using. I would name it selectedBoardName. Usually for this purpose id s are used. Are the board names unique? If so this can also work fine.

Reply:
done